com.microsoft.mrtk.core/Editor/Inspectors/PropertyDrawers/VariableRangeAttributeDrawer.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Editor
+{
+    [CustomPropertyDrawer(typeof(VariableRangeAttribute))]
+    public class VariableRangePropertyDrawer : PropertyDrawer
+    {
+        private GUIStyle labelStyle;
+
+        public VariableRangePropertyDrawer()
+        {
+            labelStyle = EditorGUIUtility.GetBuiltinSkin(EditorSkin.Inspector).label;
+        }
+
+        public override void OnGUI(Rect position, SerializedProperty property, GUIContent label)
+        {
+            EditorGUI.BeginProperty(position, label, property);
+
+            VariableRangeAttribute range = attribute as VariableRangeAttribute;
+            if (range == null)
+            {
+                Debug.LogError($"Property was did not have a VariableRangeAttribute");
+                return;
+            }
+
+            string minVariablePath = property.propertyPath.Contains(".") ? System.IO.Path.ChangeExtension(property.propertyPath, range.MinVariableName) : range.MinVariableName;
```

Comment:
Huh, for my understanding, what is this logic with the `ChangeExtension` call checking for and doing?

Reply:
Removed, it was vestigial code from another attribute drawer (removed it there too!)